documentation/breaking-changes/breaking-changes-attribute-help.md

```diff
@@ -0,0 +1,401 @@
+# Breaking Changes Attribute Help
```

Comment:
I think we should start with something more prescriptive , with a reference to the breaking change policy before we get to technical docs on the attribute.  Also, I think we should call this document 'about-breaking-changes'

Reply:
In general, this does an outstanding job of describing what the attributes do, but is a lot of readin if what I want to know is - how do I specify breaking changes of various types using the attributes.  So I would organize this around the types of breaking changes cmdlets can have, and how the attribute would be used in that case (with the nice parts about runtime effects and the like in with the descriptions)

Reply:
@markcowl 
We have [this document](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/27) that already captures the types of breaking changes... As of now its a help file for the tool, but I think we can re-organize it to say what attribute should be used to call out that particular type of deprecation (and a link back to the particular attribute in the breaking change attribute help doc). 

The attribute help doc can then be this small doc that only lists the four types of attributes and what the behavior is. That way the document will be short and wont necessarily have repetitive information in it.

I can get started on re-purposing the breaking change tools help doc to capture the new information. 

Let me know what your thoughts are.